src/admin-views/settings/tickets-commerce/stripe/connect/active/stripe-status.php

```diff
@@ -27,28 +27,27 @@
 ?>
 <div class="tec-tickets__admin-settings-tickets-commerce-gateway-connected-row">
 	<div class="tec-tickets__admin-settings-tickets-commerce-gateway-connected-col1">
-		<?php esc_html_e( 'Stripe Status:', 'event-tickets' ); ?>
+		<?php esc_html_e( 'Payments status:', 'event-tickets' ); ?>
 	</div>
 	<div class="tec-tickets__admin-settings-tickets-commerce-gateway-connected-col2">
-		<span class="tec-tickets__admin-settings-tickets-commerce-gateway-connect-text--connected">
-			<?php esc_html_e( 'Connected' ); ?> <span class="dashicons dashicons-saved"></span>
-		</span><br>
-		<span class="tec-tickets__admin-settings-tickets-commerce-gateway-connect-text--connected">
-			<?php esc_html_e( 'Charges Enabled' ); ?><span
-					class="dashicons <?php echo esc_attr( $charges_class ); ?>"></span>
-		</span>
-
 		<?php if ( ! empty( $capabilities ) && is_array( $capabilities ) ) : ?>
-			<h3><?php echo esc_html_e( 'Capabilities:', 'event-tickets' ); ?></h3>
 			<ul>
 				<?php foreach ( $capabilities as $capability => $status ) :
-					$capability_class = 'dashicons-yes';
+					$capability_classes = [
+						'dashicons',
+						'dashicons-yes',
+						'tec-tickets__admin-settings-tickets-commerce-gateway-capability--yes',
+					];
```

Comment:
We could avoid the if and set the values for classes dependent on bools. For example:

```suggestion
					$capability_classes = [
						'dashicons',
						'dashicons-yes' => 'inactive' !== $status,
						'tec-tickets__admin-settings-tickets-commerce-gateway-capability--yes' => 'inactive' !== $status,
						'dashicons-no' => 'inactive' === $status,
						'tec-tickets__admin-settings-tickets-commerce-gateway-capability--no' => 'inactive' === $status
					];
```

Reply:
I keep forgetting about that feature. Thank you!